lib/src/util/console_log_utils.dart

```diff
@@ -152,3 +154,51 @@ class ConsoleConfig {
   /// Captures calls to `console.log`, `console.warn`, and `console.error`.
   static const ConsoleConfig all = ConsoleConfig._('all');
 }
+
+final formatRegExp = RegExp('%[sdj%]');
+String format(dynamic f, List<dynamic> arguments) {
+  if (f is! String) {
+    final objects = [];
+    for (var i = 0; i < arguments.length; i++) {
+      objects.add(arguments[i]);
+    }
+    return objects.join(' ');
```

Comment:
```suggestion
    return [f, ...arguments].join(' ');
```

Reply:
I didn't pop the previous arguments off mate

Reply:
What do you mean?

Reply:
![i dont know](https://c.tenor.com/W3ermhPONfkAAAAM/gir-invader.gif)